app/api/groups/get_team_descendants.feature

```diff
@@ -0,0 +1,358 @@
+      | 11              | 16           | 0       |
+      | 11              | 17           | 0       |
+      | 11              | 18           | 0       |
+      | 11              | 51           | 0       |
+      | 11              | 53           | 0       |
+      | 11              | 55           | 0       |
+      | 11              | 59           | 0       |
+      | 11              | 63           | 0       |
+      | 11              | 65           | 0       |
+      | 11              | 67           | 0       |
+      | 12              | 12           | 1       |
+      | 13              | 13           | 1       |
+      | 13              | 14           | 0       |
+      | 13              | 15           | 0       |
+      | 13              | 51           | 0       |
+      | 13              | 53           | 0       |
+      | 13              | 55           | 0       |
+      | 13              | 61           | 0       |
+      | 13              | 63           | 0       |
+      | 13              | 65           | 0       |
+      | 13              | 69           | 0       |
+      | 14              | 14           | 1       |
+      | 14              | 51           | 0       |
+      | 14              | 53           | 0       |
+      | 14              | 55           | 0       |
+      | 15              | 15           | 1       |
+      | 15              | 61           | 0       |
+      | 15              | 63           | 0       |
+      | 15              | 65           | 0       |
+      | 16              | 16           | 1       |
+      | 16              | 63           | 0       |
+      | 16              | 65           | 0       |
+      | 16              | 67           | 0       |
+      | 20              | 20           | 1       |
+      | 20              | 21           | 0       |
+      | 21              | 21           | 1       |
+      | 22              | 1            | 0       |
+      | 22              | 11           | 0       |
+      | 22              | 12           | 0       |
+      | 22              | 13           | 0       |
+      | 22              | 14           | 0       |
+      | 22              | 15           | 0       |
+      | 22              | 16           | 0       |
+      | 22              | 17           | 0       |
+      | 22              | 18           | 0       |
+      | 22              | 22           | 1       |
+      | 22              | 51           | 0       |
+      | 22              | 53           | 0       |
+      | 22              | 55           | 0       |
+      | 22              | 59           | 0       |
+      | 22              | 61           | 0       |
+      | 22              | 63           | 0       |
+      | 22              | 65           | 0       |
+      | 22              | 67           | 0       |
+      | 22              | 69           | 0       |
+
+  Scenario: Get descendant teams
+    Given I am the user with ID "1"
+    # here we fixate time_spent even if it depends on NOW()
```

Comment:
I think it is not useful to add it at every usage of the step. Maybe you should just put it in the step definition if you think it is useful.

Reply:
removed DB time freezing (it is not needed here)